editor.txt

```diff
@@ -0,0 +1,107 @@
+Editor has two distinct modes accessed via command line parameters:
+- System editor used for star system editing
+- Model viewer used for model inspection
+Without any parameters, the editor will not start.
+	
+		System editor
+		~~~~~~~~~~~~
+		
+This is a gui tool for editing and authoring json formatted star systems.
+
+Usage:
+~~~~~~
+editor -system
```

Comment:
`editor --system` - while it might technically function with a single hyphen here (for backwards compatibility with the modelviewer flag), POSIX style requires the use of two hyphens when the flag contains more than a single character and thus I make no guarantees this remains compatible.